addons/better-featured-project/userstyle.css

```diff
@@ -142,3 +142,8 @@
 .sa-user-id {
   color: white;
 }
+.location {
+  word-break: break-all;
+  width: 600px;
+  margin-left: 15px;
```

Comment:
The `margin-left` makes it look wrong if scratchr2 is not enabled:

![image](https://user-images.githubusercontent.com/17484114/156445228-6c2f6c64-474f-46ea-b0ba-62a99a94087c.png)

Reply:
Okay, so I will create another CSS files that works when this addon is enabled...